pom.xml

```diff
@@ -3,8 +3,8 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.nuxeo</groupId>
-    <artifactId>nuxeo-ecm</artifactId>
-    <version>11.2-SNAPSHOT</version>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>11.3-SNAPSHOT</version>
   </parent>
 
   <groupId>org.nuxeo.ecm.core</groupId>
```

Comment:
The project is still in `11.2-SNAPSHOT`, we may want to change its versioning to not have a misunderstanding with the platform version.